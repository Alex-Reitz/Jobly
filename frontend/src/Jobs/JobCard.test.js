import React from "react";
import { render } from "@testing-library/react";
import Card from "./JobCard";
import { MemoryRouter } from "react-router";
import UserContext from "../Context/userContext";

it("matches snapshot", function () {
  const testFunc = jest.fn();
  const set1 = new Set();
  const info = jest.fn(() => 123);
  const { asFragment } = render(
    <MemoryRouter>
      <UserContext.Provider
        value={{ currentUser: testFunc, applicationIds: set1 }}
      >
        <Card info={info} />
      </UserContext.Provider>
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
