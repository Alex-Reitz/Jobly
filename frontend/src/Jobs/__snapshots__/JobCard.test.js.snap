// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="job-content"
  >
    <div
      class="company-title"
    />
    <div
      class="job-title-card"
    />
    <div
      class="job-salary-card"
    >
      <p>
        Sorry, no salary data to show.
      </p>
    </div>
    <div
      class="job-equity-card"
    >
      <p>
        Sorry, no equity data to show.
      </p>
    </div>
    <div
      class="job-application-card"
    >
      <button
        class="job-card-apply"
      >
        Apply
      </button>
    </div>
  </div>
</DocumentFragment>
`;
